stackslib/src/net/download/nakamoto/tenure_downloader_set.rs

```diff
@@ -407,19 +482,43 @@ impl NakamotoTenureDownloaderSet {
                 continue;
             };
 
+            let attempt_count = if let Some(attempt_count) = self.attempted_tenures.get(&ch) {
+                *attempt_count
+            } else {
+                0
+            };
+            self.attempted_tenures
+                .insert(ch.clone(), attempt_count.saturating_add(1));
+
+            let attempt_failed_count =
+                if let Some(attempt_failed_count) = self.attempt_failed_tenures.get(&ch) {
+                    *attempt_failed_count
+                } else {
+                    0
+                };
+
             info!("Download tenure {}", &ch;
```

Comment:
This is a nit, but do you think in general we could start putting vars inline where possible as a general coding standard? :D not important by any means but it makes things much easier to read when you see "Download tenure {&ch}" vs what is here currently.